mimic/rest/maas_api.py

```diff
@@ -118,7 +118,7 @@ def create_entity(params):
     """
     params = collections.defaultdict(lambda: '', params)
     newentity = {}
-    newentity['label'] = params[u'label'].encode("utf-8")
+    newentity['label'] = params['label']
```

Comment:
"strict ASCII" isn't a Python data type though :).  Sorry, what I meant to say was "`unicode`".

In Python, there are 3 string types; two that you know about and one secret one.

The first is "bytes", which is `bytes` on python 2 ("`str`" is an alias) and `bytes` on python 3.  This represents binary data, which may of course represent text, but it implies a pile of octets that you haven't fully decided upon the meaning of yet.

The second is "text", which is `unicode` on python 2 and `str` on python 3 - sometimes referred to as `six.text_type` by users of the `six` library for writing code that works on 2 and 3 simultaneously.

The final, secret string type is "native string", which is `str` on python 2 and `str` on Python 3.  Certain interpreter internals, like identifiers and docstrings, are represented with `str` on python 2, but are unambiguously text as far as the user is concerned.  This is just terrible, but since they're almost always ASCII you can usually get along without suffering too much.

In this case, the important thing about the return value is that it's "`dict` mapping `unicode` to `unicode`".  The fact that the keys are ASCII-only `unicode` objects doesn't change the fact that they are still text keys.

Also, Mimic uses ReStructuredText for its docstrings, which means rather than "this function returns…" you can say something like this instead:

```rest
:return: an Entity model, which is described in `the Rackspace Cloud Monitoring
    Developer Guide, section 5.4
    <http://docs.rackspace.com/cm/api/v1.0/cm-devguide/content/service-entities.html>`_
:rtype: ``dict`` mapping ``unicode`` to ``unicode``, ``list``, or ``dict``.
```